appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -1224,6 +1223,117 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(bob)
 	}
 
+	def 'Richard and Bob have different goals'()
```

Comment:
Agreed. At the time I started the test, I didn't have a clear idea what the test would do.

I'll rename it in "Richard's goals are only included when a buddy has that goal too"